docs: visualize-data (#211)
GreptimeTeam/docs: docs/getting-started/visualize-data.md

# Visualize data

## Introduction

Visualization is a very important part of the use of time series data.

> Greptime provides a simple [dashboard](https://github.com/GreptimeTeam/dashboard) to help users take advantage of GreptimeDB's various features.

Reviewer: ```suggestion
To help users leverage the various features of GreptimeDB, Greptime offers a simple [dashboard](https://github.com/GreptimeTeam/dashboard).
```

> Current version supports MySQL and Python queries (support for PromQL coming soon, stay tuned)

Reviewer: ```suggestion
The current version of the dashboard supports MySQL and Python queries, with support for PromQL coming soon. Please stay tuned for updates.
```

![](../public/dashboard-select.jpg)

> For different scenarios, we provide different chart types to choose from.

Reviewer: ```suggestion
We offer various chart types to choose from based on different scenarios.
```

![line](../public/dashboard-line.jpg)
![scatter](../public/dashboard-scatter.jpg)

> We will continue to develop and iterate this open source project, and will increase the application of time series data in monitoring, analysis and other fields in the future.

Reviewer: ```suggestion
We are committed to the ongoing development and iteration of this open source project, and we plan to expand the application of time series data in monitoring, analysis, and other relevant fields in the future.
```
